README.md

```diff
@@ -1,52 +1,37 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Leaderboard
 
-> One paragraph statement about the project.
+> CHANGE IT
 
-![screenshot](./app_screenshot.png)
+![screenshot](./screenshot.png)
```

Comment:
- [ ] It looks like you have an issue with your image. I kindly invite you to fix it. 🐶 
![image](https://user-images.githubusercontent.com/20563252/128035490-2f0e0120-4fb8-44f4-bf36-52fcca44358e.png)

Reply:
This is first milestone-1 project is not completed in this milestone so I didn't add it. But I will add. Thank you.